### ISSUE TITLE
Reshaping in .Rmd documents

### ISSUE THREAD

**Issue by luerhard** [lionel-/codegrip#7]
Hi,
I just tested this and it works great!

Nevertheless, I only got it working in .R documents (R Scripts) and not in .Rmd (R Markdown). 

Am I doing something wrong or is this not implemented (yet)? If so, is an implementation for R Markdown planned?

Keep up the great work and many thanks!

**Comment by jobrachem** [comment]
I agree. I love the addin, and would really like to see it working in .Rmd and .qmd files as well 😊

**Comment by olivroy** [comment]
+1, also maybe just in source mode for now, since visual mode may be more tricky

**Comment by JosiahParry** [comment]
+1

**Comment by rmtrane** [comment]
Not sure what the status of this project is, but absolutely love it and use it all the time. For a while, I've been slightly annoyed when going from .R files to .qmd/.Rmd files and find alt+tab doesn't invoke codegrip. I finally had some time to hack something together that works. If anyone else is curious, see my forked repo here: https://github.com/rmtrane/codegrip. `remotes::install_github("rmtrane/codegrip")` should give you the ability to reshape in .Rmd/.qmd files. 

Note: this has not be thoroughly tested, but has worked well for me so far.
